### PR TITLE
#155965583 Build a FAQ with live chat and help videos

### PR DESCRIPTION
**Pull request by frankip**
#### What does this PR do?
It integrates the Live chat widget that post the questions to slack while also adding an FAQ model and page 
#### How should this be manually tested?
on the  bottom right there is a green button for chat, click on it and type your question, it should reflect on the slack channel join `https://join.slack.com/t/bigas-opus/signup`the go to `Live Chat` channel.
Also on the on the navigation bar go to the FAQ page and you'll find FAQ's with relevant videos.
screen shot 2018-04-05 at 19 19 05
## How the chat looks 
screen shot 2018-04-05 at 19 21 04

### CODE REVIEW

File: hc/front/models.py [hunk 1]
```diff
@@ -1,3 +1,12 @@
 from django.db import models
 
 # Create your models here.
+
+class FaqQuestions(models.Model):
```
Comment (meshnesh) [inline hc/front/models.py:5]: Nice implementation of model function

File: templates/front/help_center.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                <h4>
+                    <strong>Answer</strong>
+                </h4>
+                <p class="mb-0">{{faq.answer }}</p>
+            </div>
+            <div class="col-6 col-md-6">
+                <h5>link</h5>
+                    <P class="alert-link">{{ faq.link|default:"No video" }}</P>
+                <div class="embed-responsive embed-responsive-16by9">
+                    <!-- <iframe width="400" height="315" src="https://www.youtube.com/embed/CTFtOOh47oo" frameborder="0" allow="autoplay; encrypted-media"
```
Comment (meshnesh) [inline templates/front/help_center.html]: remove the commented code, if it's not being used

File: hc/front/views.py [hunk 3]
```diff
@@ -565,3 +567,13 @@ def privacy(request):
 
 def terms(request):
     return render(request, "front/terms.html", {})
+
+def help_center(request):
```
Comment (meshnesh) [inline hc/front/views.py:571]: good implementation

**Review by meshnesh — COMMENTED** [review]
nice implementation of the FAQ feature

**Comment by mutoromike** [comment]
Great implementation of the chat functionality!!

**Pull request merged**